plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js

```diff
@@ -62,7 +62,11 @@ function createFile(currentTaskName, state, taskNode) { //createFile
     var outputElements = $('#' + currentTaskName + 'wrapper #outputmappingTable tr');
     $.get('resources/template.wsdl', function(data) {
        try {
-            wsdlDom = marshalEditorTextContent(data);
+            if (typeof data === "string"){
+               wsdlDom = marshalEditorTextContent(data);
+            }else {
```

Comment:
Fixed in https://github.com/wso2/devstudio-tooling-bps/pull/18/commits/fbd39184fe68f88389a0dd836a98dcfa938b1900